dbt/seeds/ccao/ccao.class_dict.csv

```diff
@@ -14,11 +14,11 @@ major_class_code,major_class_type,regression_class,modeling_group,reporting_grou
 2,Residential,TRUE,SF,Single-Family,2,208,"Two or more story residence, up to 62 years, 3,801 to 4,999 sq. ft.",0,62,3801,4999
 2,Residential,TRUE,SF,Single-Family,2,209,"Two or more story residence, any age, 5,000 sq. ft. and over",0,Infinity,5000,Infinity
 2,Residential,TRUE,SF,Single-Family,2,210,"Old style townhouse, over 62 years",63,Infinity,0,Infinity
-2,Residential,TRUE,MF,Multi-Family,2,211,"Two to six residential apartments, any age",0,Infinity,0,Infinity
+2,Residential,TRUE,MF,Multi-Family,2,211,"Apartment building with 2 to 6 units, any age",0,Infinity,0,Infinity
```

Comment:
This language is a 1:1 reflection of the pdf document, although I'm not sure if it made sense to change this. Lmk what you think @ccao-jardine